xml/art_sle_ha_pmremote.xml

```diff
@@ -115,13 +144,14 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Remote Node</term>
+     <term>Physical Remote Node</term>
```

Comment:
Hmm, to avoid confusion, I'm now kind of inclined to keep this consistent with upstream to use the term "Remote Node" everywhere in our doc...